### DISCUSSION TITLE
AnyLabeling v0.2.12

### DISCUSSION THREAD

**Discussion by vietanhdev** [vietanhdev/anylabeling#59] [Announcements]
## What's Changed

- Update icons.
- Fixed: Could not open single image file.

If you find this project useful, please consider [sponsoring](https://ko-fi.com/vietanhdev) its development.

<hr /><em>This discussion was created from the release <a href='https://github.com/vietanhdev/anylabeling/releases/tag/v0.2.12'>AnyLabeling v0.2.12</a>.</em>

**Comment by ZinouAMZ** [comment]
hello,
when i chose the model the app blocked and then closed, any solution please ?
thanks

**Reply by vietanhdev** [reply]
Have you restarted it and try again?
Please provide some more information to help us debug more easily:
- Your error message
- Your operating system
- Are you behind a proxy?
- Your PC hardware if possible
- The version of AnyLabeling you are using
Thank you!
